src/presentation/http/http-server.ts

```diff
@@ -51,6 +59,17 @@ export default class HttpServer implements API {
      */
     this.server.register(NoteRouter, { prefix: '/note',
       noteService: domainServices.noteService });
+
+    /**
+     * Check if environment is development
+     */
+    if (this.environment === 'development') {
```

Comment:
maybe it would be better to add `allowCors` options to the `HttpApiConfig` with values `false`, `*` or custom ?